plugins/api-docs/src/components/ApiExplorerPage/DefaultApiExplorerPage.tsx

```diff
@@ -103,7 +106,9 @@ export const DefaultApiExplorerPage = (props: DefaultApiExplorerPageProps) => {
               to={registerComponentLink?.()}
             />
           )}
-          <SupportButton>All your APIs</SupportButton>
+          <SupportButton>
```

Comment:
Could we translate the above create button title?

Reply:
Sure